guides/common/modules/snip_red-hat-helper-apps.adoc

```diff
@@ -0,0 +1,11 @@
+ifdef::upgrading-connected,upgrading-disconnected[]
+For interactive upgrade instructions, you can use the {ProjectName} Upgrade Helper on the Red{nbsp}Hat Customer Portal.
+This application provides you with an exact guide to match your current version number.
+You can find instructions that are specific to your upgrade path, as well as steps to prevent known issues.
+For more information, see https://access.redhat.com/labs/satelliteupgradehelper/[{Project} Upgrade Helper] on the Red{nbsp}Hat Customer Portal.
+endif::[]
+ifdef::installing-satellite-server-connected,installing-satellite-server-disconnected,installing-capsule-server[]
+For interactive installation instructions, you can use the {ProjectName} Installation Helper on the Red{nbsp}Hat Customer Portal.
+This application provides you with an interactive way to prepare installation instructions to match your required {Project} version number and configuration.
```

Comment:
```suggestion
This application provides you with an interactive way to prepare installation instructions for obtaining your required {Project} version number and configuration.
```

Reply:
Updated, but not exactly as you suggested. I went with a sentence that uses the word "customized" here too.